static/templates/user_info_popover_content.handlebars

```diff
@@ -28,6 +28,12 @@
         {{/if}}
     </div>
     <hr />
+    <li>
+        <div class="copy_email" data-clipboard-action="copy" data-clipboard-text="{{ user_email }}">
+            <i class="fa fa-clone" aria-hidden="true"></i>
+            <span>Copy user's email</span>
```

Comment:
Any user-facing strings like this must be be tagged for translation.  See zulip.readthedocs.io/en/latest/translating/index.html

Reply:
Fixed